web/src/main/java/com/navercorp/pinpoint/web/service/map/processor/RpcCallProcessor.java

```diff
@@ -109,6 +110,29 @@ public LinkDataMap processLinkDataMap(LinkDataMap linkDataMap, Range range) {
         return Collections.singletonList(linkData);
     }
 
+    private Set<AcceptApplication> filterAlias(Set<AcceptApplication> acceptApplicationList) {
+
+        if (acceptApplicationList.size() < 2) {
+            return acceptApplicationList;
+        }
+
+        final Set<AcceptApplication> resultSet = new HashSet<>();
+
+        for (AcceptApplication acceptApplication : acceptApplicationList) {
+            if (!acceptApplication.getApplication().getServiceType().isAlias()) {
+                resultSet.add(acceptApplication);
+            } else {
+                logger.debug("deduct alias application {}", acceptApplication);
+            }
+        }
+
+        if (resultSet.size() == 0) {
```

Comment:
Would `isEmpty()` suffice?

Reply:
I don't understand what you're suggesting.
`filterAlias` method is trying to remove aliased application types if there are application types from non-alias serviceTypes. 
This logic is used when there are both client-plugin and server-plugin

Reply:
Nothing so extensive, was suggesting for clarity, `if (resultSet.size() == 0) {` -> `if (resultSet.isEmpty()) {`

Reply:
oh, got it